tests/ci/azure_pipeline_test/dsvm_notebook_linux_cpu_.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
small error in the name, there is an extra underscore

Reply:
@yueguoguo do you know if fixing the underscore needs to set up the pipeline again or just by renaming it should work?

Reply:
We need to point to the new pipeline yaml files. We need to do it anyway after everything is merged to master branch - currently the yaml files for the testing pipeline are those in the `le_test_new` branch, temporarily.